libethereum/Client.cpp

```diff
@@ -613,10 +613,11 @@ bool Client::submitWork(ProofOfWork::Solution const& _solution)
 
 void Client::syncBlockQueue()
 {
-	ImportRoute ir;
 	cwork << "BQ ==> CHAIN ==> STATE";
-	tie(ir.first, ir.second, m_syncBlockQueue) = m_bc.sync(m_bq, m_stateDB, rand() % 10 + 5);
```

Comment:
make `m_bc.sync` return a `pair<ImportRoute, bool>` to avoid this additional code